documentation/synthetic_tokens/tutorials/creating_from_truffle.md

```diff
@@ -144,30 +144,30 @@ await emp.positions(accounts[0]);
 ```js
 await emp
   .deposit({ rawValue: web3.utils.toWei("10") })(await collateralToken.balanceOf(accounts[0]))
-  .toString();
+  .toString()
 ```
 
 2. For a token sponsor to withdraw collateral from his position, there are typically 2 ways to do this.
    Read this [explainer](../explainer.md) for more information.
    In this scenario, because we are the only token sponsor, we will have to withdraw collateral the “slow” way. First, we need to request a withdrawal of 10 collateral tokens.
 
 ```js
-await emp.requestWithdrawal({ rawValue: web3.utils.toWei("10") });
+await emp.requestWithdrawal({ rawValue: web3.utils.toWei("10") })
 ```
 
 3. Now, we need to simulate the withdrawal liveness period passing without a dispute of our withdrawal request.
    Because we set this period to be 1000 when we parameterized and deployed the contract, let’s simulate 1001 of time elapsing.
    Once the 1001 has elapsed, we can withdraw the amount we previously requested.
 
 ```js
-await emp.setCurrentTime((await emp.getCurrentTime()).toNumber() + 1001);
-await emp.withdrawPassedRequest();
+await emp.setCurrentTime((await emp.getCurrentTime()).toNumber() + 1001)
+await emp.withdrawPassedRequest()
 ```
 
 4. Let’s now check that our collateral token balance has returned to 9,925.
 
 ```js
-(await collateralToken.balanceOf(accounts[0])).toString();
+;(await collateralToken.balanceOf(accounts[0])).toString()
```

Comment:
nit: is there supposed to be a semicolon at the beginning of the line?

Reply:
So strange. if you have double brackets with one around the `await` and then cast the output to string it throws this error. I've updated this by storing the state in a variable between points and it passes now without adding the `;`

Reply:
Weird :/